Keypad/Keypad.js

```diff
@@ -302,7 +335,7 @@ const KeypadBehaviorDecorator = hoc((config, Wrapped) => {
 					// do nothing;
 					break;
 
-				case 'phone':
+				case 'phone' || 'callaccept':
```

Comment:
I think it does not work.
```suggestion
				case 'phone':
				case 'callaccept':
```